tests/pipeline/test_pipeline.py

```diff
@@ -1657,9 +1658,16 @@ def test_remove_pending_packages() -> None:
     os.environ["EXCEPTION_PROB"] = "1.0"
     os.environ["FAIL_IN_INIT"] = "False"
     os.environ["TIMEOUT"] = "1.0"
-    # should produce partial loads
+    # will make job go into retry state
     with pytest.raises(PipelineStepFailed):
         pipeline.run(airtable_emojis())
+    # move job into running folder manually
```

Comment:
in the new loader implementation, jobs that retry are moved into the new jobs folder again, I'm wondering wether "has_pending_data" should also return true if there are jobs that are marked as "to be retried"

Reply:
@rudolfix I need your opinion here.